File: joseffe-barroso/Python/aula03/exercise22.py
choice = int(input('''Informe seu estado civil
    1 - Triângulo
    2 - Quadrado
    3 - Retângulo
    4 - Circulo
    5 - FIM DE PROCESSO (sair do programa)

    Digite o número correspondente: '''))

if (choice == 1 or choice == 2 or choice == 3 or choice == 4):

  if (choice == 1):
    triangleHeight = float(input('Digite a altura do triângulo: '))
    triangleBase = float(input('Digite a base do triângulo: '))
    totalArea = (triangleBase * triangleHeight) / 2
    print("A área do triângulo é: ", totalArea)

  elif (choice == 2):
    squareSide = float(input('Digite o lado do quadrado: '))
    totalArea = squareSide * squareSide
    print("A área do quadrado é: ", totalArea)

  elif (choice == 3):
    rectangleHeight = float(input('Digite a altura do retângulo: '))
    rectangleBase = float(input('Digite a base do retângulo: '))
    totalArea = rectangleBase * rectangleHeight
    print("A área do retângulo é: ", totalArea)

  elif (choice == 4):
    circleRadius = float(input('Digite o raio do círculo: '))

    pi = 3.14
    totalArea = pi * circleRadius**2
    print("A área do círculo é: ", totalArea)
  
  
elif (choice == 5):
  print("Até mais")

else:
  print("Informe um valor que que coincida na lista")

File: joseffe-barroso/Java/src/aula03/ddd/Exercise22.java
package ddd;

import java.util.Scanner;

public class Exercise22 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double triangleHeight, triangleBase, squareSide, rectangleHeight, rectangleBase, circleRadius, totalArea;
		int choice;

		System.out.println("Cálculo de área, digite o número da opção correspondente");
		System.out.println("1 - Triângulo");
		System.out.println("2 - Quadrado");
		System.out.println("3 - Retângulo");
		System.out.println("4 - Circulo");
		System.out.println("5 - FIM DE PROCESSO (sair do programa)");

		System.out.printf("Digite o número da operação:");
		choice = scanner.nextInt();

		if (choice == 1 || choice == 2 || choice == 3 || choice == 4 || choice == 5) {
			if (choice == 5) {
				System.out.println("Até mais");
				scanner.close();
				return;
			} else if (choice == 1) {
				// Triângulo
				System.out.printf("Digite a altura do triângulo: ");
				triangleHeight = scanner.nextDouble();

				System.out.printf("Digite a base do triângulo: ");
				triangleBase = scanner.nextDouble();

				totalArea = (triangleBase * triangleHeight) / 2;

				System.out.printf("A área do triângulo é: %f", totalArea);

			} else if (choice == 2) {
				// Quadrado
				System.out.printf("Digite o lado do quadrado: ");
				squareSide = scanner.nextDouble();

				totalArea = squareSide * squareSide;

				System.out.printf("A área do quadrado é: %f", totalArea);

			} else if (choice == 3) {
				// Retângulo
				System.out.printf("Digite a altura do retângulo: ");
				rectangleHeight = scanner.nextDouble();

				System.out.printf("Digite a base do retângulo: ");
				rectangleBase = scanner.nextDouble();

				totalArea = rectangleBase * rectangleHeight;

				System.out.printf("A área do retângulo é: %f", totalArea);

			} else if (choice == 4) {
				// Circulo
				System.out.printf("Digite o raio do círculo: ");
				circleRadius = scanner.nextDouble();

				double pi;
				pi = 3.14;

				totalArea = pi * Math.pow(circleRadius, 2);

				System.out.printf("A área do círculo é: %f", totalArea);

			}
		}

		scanner.close();
	}

}
